charts/posthog/values.yaml

```diff
@@ -592,6 +594,29 @@ clickhouse:
     shardsCount: 1
     replicasCount: 1
 
+  ## -- ClickHouse settings configuration.
+  ## You can use this to override settings, for example `prometheus/port: 9363`
+  ## For the full list of settings, see:
+  ## - https://clickhouse.com/docs/en/operations/settings/settings/
+  settings: {}
+    # Uncomment those lines if you want to enable the built-in Prometheus HTTP endpoint in ClickHouse.
+    # prometheus/endpoint: /metrics
+    # prometheus/port: 9363
+    # prometheus/metrics: true
+    # prometheus/events: true
+    # prometheus/asynchronous_metrics: true
+
+  ## -- Default settings configuration for ClickHouse. Don't overwrite this!
+  defaultSettings:
```

Comment:
I think this is a valid point, let's discuss! Specifically to this implementation, I've used the same pattern we already have in use.

Reply:
Right yeah I see. I can't quite see what the advantage of having `defaultSettings` AND an empty `settings` rather than just a `settings` with the value specified but maybe I'm missing something.

Anyway, I don't think there's a reason to block on this.

Reply:
I think the idea was to have a place where settings can be easily added (e.g. via Helm CLI) without having to port over a lot of default entries + the one you are adding. 

@macobo is that the reason behind it?

Reply:
You are correct. Having this separation helps if we require some settings but others which users can override.